homeassistant/components/config/entity_registry.py

```diff
@@ -240,6 +240,7 @@ def _entry_dict(entry: er.RegistryEntry) -> dict[str, Any]:
         "name": entry.name,
         "original_name": entry.original_name,
         "platform": entry.platform,
+        "unique_id": entry.unique_id,
```

Comment:
If this is added here, it should be removed from `_entry_ext_dict` below

Reply:
Hey @emontnemery, thanks for your feedback! I had picked the "unique_id" field as this is also provided as part of the update response. If we would introduce the RegistryEntry.id (which make sense), other responses need to be adapted as well.

The main use case is to identify entities after the entity_id has been changed. I am developing a HA Addon UI that will allow power users to edit registry entries faster (advanced filtering, inline editing, possibly mass updates, etc). For these use cases I needed a constant/reliable UID to keep my EntityCache in sync with the EntityRegistry.